build_tools/github_actions/determine_version.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python
+"""Determines the SDK version and version suffix to pass as additional
+arguments to `external-builds/pytorch/build_prod_wheels.py`.
+
+Example usage:
+
+    python determine_version.py --rocm-version 7.0.0 --write-env-file
+"""
```

Comment:
Can you include what the expected output and side effects from running the script are in this comment? Looks like it writes `optional_build_prod_arguments` to `GITHUB_ENV`